From 196681c8eb287df7dd2487f6bd264c09d27b89b3 Mon Sep 17 00:00:00 2001
From: Yang Jeong Hun <onyxclover9931@gmail.com>
Date: Tue, 16 Jul 2024 07:22:49 +0900
Subject: [PATCH] drivers: hv: dxgkrnl: Add missing include vmalloc header file

 * Add missing <linux/vmalloc.h> header file to use vzalloc() and vfree().

Signed-off-by: Yang Jeong Hun <onyxclover9931@gmail.com>
---
 drivers/hv/dxgkrnl/dxgadapter.c | 1 +
 drivers/hv/dxgkrnl/dxgvmbus.c   | 1 +
 drivers/hv/dxgkrnl/hmgr.c       | 1 +
 drivers/hv/dxgkrnl/ioctl.c      | 1 +
 4 files changed, 4 insertions(+)

diff --git a/drivers/hv/dxgkrnl/dxgadapter.c b/drivers/hv/dxgkrnl/dxgadapter.c
index 6d3cabb24e6f3..0844ef064a4e3 100644
--- a/drivers/hv/dxgkrnl/dxgadapter.c
+++ b/drivers/hv/dxgkrnl/dxgadapter.c
@@ -15,6 +15,7 @@
 #include <linux/hyperv.h>
 #include <linux/pagemap.h>
 #include <linux/eventfd.h>
+#include <linux/vmalloc.h>
 
 #include "dxgkrnl.h"
 
diff --git a/drivers/hv/dxgkrnl/dxgvmbus.c b/drivers/hv/dxgkrnl/dxgvmbus.c
index abb6d2af89acc..59149b2969a46 100644
--- a/drivers/hv/dxgkrnl/dxgvmbus.c
+++ b/drivers/hv/dxgkrnl/dxgvmbus.c
@@ -19,6 +19,7 @@
 #include <linux/mman.h>
 #include <linux/delay.h>
 #include <linux/pagemap.h>
+#include <linux/vmalloc.h>
 #include "dxgkrnl.h"
 #include "dxgvmbus.h"
 
diff --git a/drivers/hv/dxgkrnl/hmgr.c b/drivers/hv/dxgkrnl/hmgr.c
index 24101d0091ab3..528402a9ee491 100644
--- a/drivers/hv/dxgkrnl/hmgr.c
+++ b/drivers/hv/dxgkrnl/hmgr.c
@@ -14,6 +14,7 @@
 #include <linux/kernel.h>
 #include <linux/mutex.h>
 #include <linux/rwsem.h>
+#include <linux/vmalloc.h>
 
 #include "misc.h"
 #include "dxgkrnl.h"
diff --git a/drivers/hv/dxgkrnl/ioctl.c b/drivers/hv/dxgkrnl/ioctl.c
index 42f3de31a63cb..d70a4b9153bce 100644
--- a/drivers/hv/dxgkrnl/ioctl.c
+++ b/drivers/hv/dxgkrnl/ioctl.c
@@ -17,6 +17,7 @@
 #include <linux/anon_inodes.h>
 #include <linux/mman.h>
 #include <linux/pid_namespace.h>
+#include <linux/vmalloc.h>
 
 #include "dxgkrnl.h"
 #include "dxgvmbus.h"
